### PR TITLE
Add document for upgrading from 1.5 to 1.6

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

upgrade from 1.5 to 1.6

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/upgrading/v1.5-v1.6.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: v1.5 升级到 v1.6
+---
+
+遵循[常规升级流程](./README.md)。
+
+## 升级后显著变化
+
+### API变化
+
+* PropagationPolicy 中  `AffinityName` 的长度现在限制为 (1, 32]，并且必须是合规名称。
```
Comment (zishen) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/upgrading/v1.5-v1.6.md:11]: c1253dbe2e7afaa81a94022f2fcd11c
It seems [1, 32], right?

Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current/administrator/upgrading/v1.5-v1.6.md]: Thanks, update.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**